app/components/confirm-box/index.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable ember/no-classic-components, ember/no-classic-classes, ember/require-tagless-components, ember/no-get */
```

Comment:
@future-pirate-king This component wasn't migrated?

Reply:
no this component is used in other places so only wanted to refactor ui and not implementation